erpnext/assets/dashboard_chart/asset_value_analytics/asset_value_analytics.json

```diff
@@ -0,0 +1,27 @@
+{
+ "chart_name": "Asset Value Analytics",
+ "chart_type": "Report",
+ "creation": "2020-07-14 18:23:53.091233",
+ "custom_options": "{\"type\": \"bar\", \"barOptions\": {\"stacked\": 1}, \"axisOptions\": {\"shortenYAxisNumbers\": 1}, \"tooltipOptions\": {}}",
+ "docstatus": 0,
+ "doctype": "Dashboard Chart",
+ "dynamic_filters_json": "{\"company\":\"frappe.defaults.get_user_default(\\\"Company\\\")\"}",
+ "filters_json": "{\"status\":\"In Location\",\"filter_based_on\":\"Fiscal Year\",\"from_fiscal_year\":\"2020-2021\",\"to_fiscal_year\":\"2020-2021\",\"period_start_date\":\"2020-04-01\",\"period_end_date\":\"2021-03-31\",\"date_based_on\":\"Purchase Date\",\"group_by\":\"--Select a group--\",\"from_date\":\"2019-07-21\",\"to_date\":\"2020-07-21\"}",
```

Comment:
from_fiscal_year, to_fiscal_year and most date filters can be added to dynamic filters. Use `frappe.sys_defaults.fiscal_year` to get dynamic value for fiscal year fields
Refer to the report `Fixed Asset Register` to see which filters are necessary.